### PR TITLE
[NoJira] update screenshots for banner alert, update button storybook

### PR DESCRIPTION
**Pull request by matteo-hertel**

### CODE REVIEW

File: native/package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "link": "react-native link",
     "start": "react-native start",
     "ios": "react-native run-ios",
-    "android:emulator": "$ANDROID_HOME/tools/emulator -avd bpk-avd -dpi-device 420 -skin 1080x1920",
+    "android:emulator": "$ANDROID_HOME/tools/emulator -avd bpk-avd",
```
Comment (matteo-hertel) [inline native/package.json]: This is change is on purpose, will prevent the Pixel2 emulator to be forced at 1080 and render the content is a weird way

Comment (shaundon) [reply native/package.json]: Oh awesome - I wrote this quite a while ago and it was more-or-less hacked together, so thanks for optimising it!

File: native/packages/react-native-bpk-component-button/stories.js [hunk 2]
```diff
@@ -46,11 +46,17 @@ const styles = StyleSheet.create({
 const getIconType = type =>
   type === 'destructive' ? icons.trash : icons['long-arrow-right'];
 
-const generateButtonStoryForType = (type: string) => {
+const generateButtonStoryForType = (
+  type: string,
+  storyAsLable: boolean = false,
```
Comment (shaundon) [inline native/packages/react-native-bpk-component-button/stories.js]: Should be `storyAsLabel`

Comment (matteo-hertel) [reply native/packages/react-native-bpk-component-button/stories.js]: 💔

**Comment by matteo-hertel** [comment]
I've fixed the typo and below a screenshot of the docs with the new scresnshots, should be good to go

screen shot 2018-02-26 at 08 45 19-fullpage

**Pull request merged**